tox.ini

```diff
@@ -4,7 +4,6 @@
 [tox]
 #          python version             test group                  extra envs  extra commands       
 envlist = {py27,py35,py36}-{flakes,unit,examples,all_recommended}-{default}-{dev,pkg}
-build = wheel
```

Comment:
I'd hope you could leave this; please let me know if things don't work with this still present.

Reply:
I'll revert this, I was trying to work around py27 issues and this was just collateral.